src/components/calcite-action/calcite-action.tsx

```diff
@@ -83,6 +83,19 @@ export class CalciteAction {
 
   @Element() el: HTMLCalciteActionElement;
 
+  private buttonEl?: HTMLButtonElement;
```

Comment:
Nitpick: I don't think you need to make this optional since it should get populated on the first render. Kind of similar to `el`, no?